applications/MeshMovingApplication/python_scripts/mesh_moving_analysis.py

```diff
@@ -16,252 +16,100 @@
-            self.list_of_processes += process_factory.KratosProcessFactory(self.model).ConstructListOfProcesses(self.ProjectParameters["json_output_process"])
-        # Processes for tests
-        if (self.ProjectParameters.Has("json_check_process") == True):
-            self.list_of_processes += process_factory.KratosProcessFactory(self.model).ConstructListOfProcesses(self.ProjectParameters["json_check_process"])
-        if (self.ProjectParameters.Has("check_analytic_results_process") == True):
-            self.list_of_processes += process_factory.KratosProcessFactory(self.model).ConstructListOfProcesses(self.ProjectParameters["check_analytic_results_process"])
-
-        if self.is_printing_rank and (self.echo_level > 1):
-            count = 0
-            for process in self.list_of_processes:
-                count += 1
-                # KratosMultiphysics.Logger.PrintInfo("Process " + str(count), process) # FIXME
-
-        ## Processes initialization
-        for process in self.list_of_processes:
-            process.ExecuteInitialize()
-
-        ## Solver initialization
-        self.solver.Initialize()
-
-    def __ExecuteBeforeSolutionLoop(self):
-        """ Perform Operations before the SolutionLoop """
-        if (self.output_post == True):
-            self.gid_output.ExecuteBeforeSolutionLoop()
-
-        for process in self.list_of_processes:
-            process.ExecuteBeforeSolutionLoop()
-
-        ## Writing the full ProjectParameters file before solving
-        if self.is_printing_rank and (self.echo_level > 1):
-            f = open("ProjectParametersOutput.json", 'w')
-            f.write(self.ProjectParameters.PrettyPrintJsonString())
-            f.close()
-
-        ## Stepping and time settings
-        self.delta_time = self.ProjectParameters["problem_data"]["time_step"].GetDouble()
-        start_time = self.ProjectParameters["problem_data"]["start_time"].GetDouble()
-        self.end_time = self.ProjectParameters["problem_data"]["end_time"].GetDouble()
-
-        if self.main_model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED] == True:
-            self.time = self.main_model_part.ProcessInfo[KratosMultiphysics.TIME]
+        return python_solvers_wrapper_mesh_motion.CreateSolver(self.model, self.project_parameters)
+
+    def _CreateProcesses(self, parameter_name, initialization_order):
+        """Create a list of Processes
+        This method is TEMPORARY to not break existing code
+        It will be removed in the future
+        """
+        list_of_processes = super(MeshMovingAnalysis, self)._CreateProcesses(parameter_name, initialization_order)
+
+        if parameter_name == "processes":
+            processes_block_names = ["boundary_conditions_process_list", "list_other_processes", "json_output_process",
+                "json_check_process", "check_analytic_results_process"]
+            if len(list_of_processes) == 0: # Processes are given in the old format
```

Comment:
Could we simply throw an error instead and cut the support for the old style?

Reply:
I'd love nothing more :D

I think in this case it is ok, since this was never really used ...

Reply:
the problem is:

what is the new format? how is it different from the "old" one? is it written somewhere?

if this is not very clear and documented somewhere, not only the old scripts do not work, i also would not be able to fix them...

Reply:
documented months ago: https://github.com/KratosMultiphysics/Kratos/wiki/Common-Python-Interface-of-Applications-for-Users#analysisstage-usage

Plus there is not even a problemtype for pure mesh-motion, currently this is only used in the tests of the MeshMotionApp

Reply:
but ... what is the change wrt the "old" format? that is what i really don't know

in any case this is "low priority" right?

If it is so, taking into account it may impact @rubenzorrilla do not merge until he gives greenlight (i think he is on holidays, so this will imply waiting a few weeks)

Reply:
old:
~~~
"boundary_processes" : [],
"initial_processes" : []
~~~
new:
~~~
"processes" : {
    "boundary_processes" : [],
    "initial_processes" : []
}
~~~

This PR does affect @rubenzorrilla , but as I wrote above I will wait for the feedback of everyone that is involved

Reply:
@msandre I will do this in a future PR, since it also requires updating of the tests